### PR TITLE
chore(deps): Update dependencies

### PR DESCRIPTION
**Pull request by Drevoed**
The lockfile and the package json have become pretty dated, I have updated the versions to the latest.

Seems to be working fine except the fact that @solid-primitives/i18n got a major breaking change. But a good one, though. Needs testing.

Tauri v2 also seems to work.

Also because the repository is missing some assets I have snatched them from revite. I hope you pardon me :).

## Please make sure to check the following tasks before opening and submitting a PR

- [x] I understand and have followed the [contribution guide](https://github.com/revoltchat/.github/blob/master/.github/CONTRIBUTING.md)
- [x] I have tested my changes locally and they are working as intended

### CODE REVIEW

File: packages/client/src/interface/Home.tsx [hunk 1]
```diff
@@ -23,12 +23,12 @@ import MdHome from "@material-design-icons/svg/filled/home.svg?component-solid";
...
 import MdRateReview from "@material-design-icons/svg/filled/rate_review.svg?component-solid";
 import MdSettings from "@material-design-icons/svg/filled/settings.svg?component-solid";
-
-import RevoltSvg from "../../public/assets/wordmark_wide_500px.svg?component-solid";
+//
+// import RevoltSvg from "../../public/assets/wordmark_wide_500px.svg?component-solid";
 
 import { HeaderIcon } from "./common/CommonHeader";
 
-const Logo = styled(RevoltSvg)`
```
Comment (insertish) [inline packages/client/src/interface/Home.tsx]: Logo is stripped out

File: packages/client/components/auth/src/flows/FlowLogin.tsx [hunk 2]
```diff
@@ -16,13 +16,13 @@ import {
...
 import MdArrowBack from "@material-design-icons/svg/filled/arrow_back.svg?component-solid";
 
-import RevoltSvg from "../../../../public/assets/wordmark_wide_500px.svg?component-solid";
+// import RevoltSvg from "../../../../public/assets/wordmark_wide_500px.svg?component-solid";
 import { clientController } from "../../../client";
 
 import { FlowTitle } from "./Flow";
 import { Fields, Form } from "./Form";
 
-const Logo = styled(RevoltSvg, {
```
Comment (insertish) [inline packages/client/components/auth/src/flows/FlowLogin.tsx]: also strips logo

File: packages/client/components/auth/src/flows/FlowHome.tsx [hunk 3]
```diff
@@ -5,9 +5,9 @@ import { useTranslation } from "@revolt/i18n";
 import { Navigate } from "@revolt/routing";
 import { Button, Column, styled } from "@revolt/ui";
 
-import RevoltSvg from "../../../../public/assets/wordmark_wide_500px.svg?component-solid";
+// import RevoltSvg from "../../../../public/assets/wordmark_wide_500px.svg?component-solid";
 
-const Logo = styled(RevoltSvg)`
```
Comment (insertish) [inline packages/client/components/auth/src/flows/FlowHome.tsx]: '' logo

**Review by insertish — CHANGES_REQUESTED** [review]
Looks good, just wondering what is going on with the logos, is this unfinished?

**Comment by Drevoed** [comment]
> Looks good, just wondering what is going on with the logos, is this unfinished?

It's a rudiment from trying to complete the build step locally. Seeing as the assets repo is out of bounds for me.

Will return now

**Comment by insertish** [comment]
Merging locally due to lock conflict.

**Comment by insertish** [comment]
Merged in 75d6c378f8add25bcfb620857699c35cfa442d54

**Pull request closed without merging**